docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1283,6 +1283,26 @@ Valid values are `-1` (unlimited), `integer` (in kilobytes), `nMB` (in megabytes
 
 Default: `1GB`
 
+## Advanced flags
+
+#### backfill_index_client_rpc_timeout_ms
+
+Timeout (in milliseconds) for the backfill stage of a concurrent CREATE INDEX.
+
+Default: 1 day
+
+#### backfill_index_timeout_grace_margin_ms
+
+The time to exclude from the YB-Master flag [ysql_index_backfill_rpc_timeout_ms](../yb-master/#ysql-index-backfill-rpc-timeout-ms) in order to return results to YB-Master in the specified deadline. Should be set to at least the amount of time each batch would require, and less than `ysql_index_backfill_rpc_timeout_ms`.
+
+Default: 1 second (auto-computed)
```

Comment:
ms so 60000 no? what does auto-computed mean?

Reply:
default is -1.  by default, that is internally computed to mean 1 second.  other flags may influence this computation, but that is complicated to explain, and this is an advanced flag anyway

Reply:
I see. Are other flag defaults, like '1 minute' also actually autocomputed, or should they be '60000 (1 minute)'